<launch>
    <node name="cvx_decomp" 
    pkg="convex_decomposer" 
    type="cvx_decomp.py" 
    output="screen">
    </node>

</launch>
